### ISSUE TITLE
Issue loading maps.

### ISSUE THREAD

**Issue by mlane48** [InvoxiPlayGames/BO3MacFix#7]
I'm having some issues loading in any of my maps that I have. This error pops up.

<img width="1340" alt="Screenshot 2024-08-17 at 4 19 44 AM" src="https://github.com/user-attachments/assets/b8fb00e1-9fda-4b8f-9881-ecc891dfb4ec">

**Comment by patricklynch7** [comment]
+1 to this error (Sonoma 14.6.1). Thank you @InvoxiPlayGames for all your hard work.

**Comment by InvoxiPlayGames** [comment]
- Does this happen with every map or just some maps? If it's every map, could you run `wine -v` in a Terminal? If only some maps, could you give me the Steam Workshop links?
- Apple Silicon or Intel mac?

**Comment by patricklynch7** [comment]
I am M3 Silicon. It's every map for me (though I only have Leviathan and Cleopatra's Lost Oasis). When I run `wine -v` I get:
fixme errors...
But more importantly:
Application could not be started, or no application associated with the specified file.
ShellExecuteEx failed: File not found.

**Comment by mlane48** [comment]
> * Does this happen with every map or just some maps? If it's every map, could you run `wine -v` in a Terminal? If only some maps, could you give me the Steam Workshop links?
> * Apple Silicon or Intel mac?

I'm running a M1 Macbook Pro using the macOS Sequoia Public Beta, running 'wine -v' gives me this error: "preloader: Warning: failed to reserve range 0000000000001000-0000000000010000
zsh: bus error  wine -v"

This error happens on every map. I don't know if I downloaded wine incorrectly or what. I really hope there is a solution to my issue.

**Comment by mlane48** [comment]
Update, I did 'wine -v' again, and it gave me more than last time.

![Screenshot 2024-08-18 at 2 11 10 PM](https://github.com/user-attachments/assets/114d7ecd-d0e3-488a-ab35-3713eff07387)

**Comment by mlane48** [comment]
Update! I went through a ton of hoops to do this, and my way was the only way I could find.

Using the Homebrew wine-stable install command in terminal, I waited to see what the installation would say on what files already existed. I deleted the file that was hidden on my macbook that existed in a hidden file and repeated the process until wine-stable was installed fully on Homebrew.

I'm glad to say that custom maps are working fine!
